tests/conformance/scripts/run-tests/index.ts

```diff
@@ -8,8 +8,11 @@ import generateTestVariants from "variants/generateTestVariants";
 import runners from "./runners";
 
 function normalizeJson(json: string): string {
-  // eslint-disable-next-line @typescript-eslint/no-unsafe-return, @typescript-eslint/no-unsafe-call
-  return stringify(JSON.parse(json));
+  // eslint-disable-next-line @typescript-eslint/no-unsafe-assignment
+  const data = JSON.parse(json);
+  // eslint-disable-next-line @typescript-eslint/no-unsafe-member-access
+  delete data.meta;
```

Comment:
Seems a little unfortunate that this `data.meta` is completely disconnected from the place where meta is added before stringifying. If the two were tied together by some type definition it could aid in debugging and changing the code later.

Reply:
Good point. Maybe once I get the writer conformance tests working I'll circle back on this.